src/mrpro/operators/LinearOperator.py

```diff
@@ -47,6 +48,59 @@ def H(self) -> LinearOperator:  # noqa: N802
         """Adjoint operator."""
         return AdjointLinearOperator(self)
 
+    def operator_norm(
+        self,
+        initial_value: torch.Tensor,
+        dim: Sequence[int] | None,
+        max_iterations: int = 64,
+    ) -> torch.Tensor:
+        """Power iteration for computing the operator norm of the linear operator.
+
+        Parameters
+        ----------
+        initial_value
+            initial value to start the iteration; if chosen exactly as zero-tensor,
```

Comment:
I really dont like this.

If I want to use a random tensor, I can create it on my own.
Maybe just throw an exception if in l.92 torch.linalg.vector_norm(vector, dim=dim, keepdim=True) is zero?